tree/CMakeLists.txt

```diff
@@ -6,12 +6,14 @@
 
 add_subdirectory(tree)
 add_subdirectory(treeplayer)
-add_subdirectory(treeviewer)
+if(TARGET Gui)
+  add_subdirectory(treeviewer)
+  add_subdirectory(ntuplebrowse)
```

Comment:
Is this line maybe redundant with the check in tree/ntuplebrowse/test/CMakeLists.txt ?